transport/src/main/java/io/netty/channel/ChannelOutboundInvokerCallback.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+/**
+ * Listener which will be notified when an {@link ChannelOutboundInvoker} methods is completed.
+ */
+// TODO: Should ChannelOutboundInvokerCallback be EventExecutor aware and so be able to delegate if needed ?
```

Comment:
I think the less state we force implementations to keep around the better.

Reply:
@chrisvest I agree.. and after thinking more about it I think it is not necessary at all.